### PR TITLE
Small cleanup of PHEx extension

### PR DESCRIPTION
**Pull request by NameIsA**
## Description
Code cleanup, re-writing manifest.

### CODE REVIEW

**Comment by NameIsA** [comment]
dont merge yet

**Comment by NameIsA** [comment]
okie you can merge

**Review by NameIsA — COMMENTED** [review]
i am cool

**Comment by BoredFishRE** [comment]
am edit manifest ok thank

**Comment by BoredFishRE** [comment]
okay just made some minor edits to the manifest things, will merge/

File: PHEx/src/background.js [hunk 1]
```diff
@@ -1,18 +1,19 @@
 const browser = chrome || browser;
-// debug will switch to redirector on localhost instead of prodigyhacking
+
+// If debug is enabled, the extension will use localhost instead of the Prodigy Hacking Organization CDN.
```
Comment (PatheticMustan) [inline PHEx/src/background.js:3]: nothing big of note, but we don't actually have a CDN

File: PHEx/src/background.js [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
 	(details) => ({ responseHeaders: details.responseHeaders.filter((header) => !["content-security-policy", "x-frame-options",].includes(header.name.toLowerCase())) }),
 	{ urls: ["<all_urls>"] }, ["blocking", "responseHeaders"]
 );
+
 // Redirect Requests
 chrome.webRequest.onBeforeRequest.addListener(details => {
 	const redirectorDomain = debug ? "http://localhost:1337" : "https://prodigyhacking.ml"
 
 	if (details.url.startsWith("https://code.prodigygame.com/code/") && details.url.includes("/game.min.js")) {
-		// if anybody curses my name, you might as well curse at me on twitter lol, I'm @PatheticMustan
```
Comment (PatheticMustan) [inline PHEx/src/background.js:15]: sad

File: PHEx/src/background.js (hunk 1 above)
Comment (BoredFishRE) [reply PHEx/src/background.js:3]: We can...

Comment (NameIsA) [reply PHEx/src/background.js:3]: > nothing big of note, but we don't actually have a CDN

whatever you call it

File: PHEx/src/background.js (hunk 2 above)
Comment (NameIsA) [reply PHEx/src/background.js:15]: i mean im fine with adding it back as long as you add a link to https://digitalteal.com too!

**Review by NameIsA — COMMENTED** [review]
i am once again cool

**Pull request merged**